oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthorisationHandler.java

```diff
@@ -709,7 +709,8 @@ private APIGatewayProxyResponseEvent generateAuthRedirect(
                         .claim("client_id", configurationService.getOrchestrationClientId())
                         .claim("redirect_uri", configurationService.getOrchestrationRedirectURI())
                         .claim("reauthenticate", reauthSub)
-                        .claim("previous_govuk_signin_journey_id", reauthSid);
+                        .claim("previous_govuk_signin_journey_id", reauthSid)
+                        .claim("channel", client.getChannel());
```

Comment:
Could you update the test for this part of the code, it's called something like `shouldPassTheCorrectClaimsToAuth`. Should be a fairly simple change 🙂

Reply:
Add new test - to test channel claim